Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      type: String
+    - contextPath: PrismaSase.SecurityRule.service
+      description: Security rule service.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source
+      description: Security rule source.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source_user
+      description: Security rule source user.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.to
+      description: Security rule to field (destination zone(s)).
+      type: String
+  - arguments:
+    - default: false
+      description: ID of the address object to edit.
+      isArray: false
+      name: object_id
+      required: true
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - default: false
+      description: The address object's description.
+      isArray: false
+      name: description
+      required: false
+    - auto: PREDEFINED
+      default: false
+      description: The type of the address.
+      isArray: false
+      name: type
+      predefined:
+      - ip_netmask
+      - ip_range
+      - ip_wildcard
+      - fqdn
+      required: false
+    - default: false
+      description: The address value (should match the type)
+      isArray: false
+      name: address_value
+      required: true
+    - default: false
+      description: A comma-separated list of address object tags.
+      isArray: true
+      name: tag
+      required: false
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Update an existing address object.
+    execution: false
+    name: prisma-sase-address-object-update
+    outputs:
+    - contextPath: PrismaSase.Address.description
+      description: Address description.
+      type: String
+    - contextPath: PrismaSase.Address.folder
+      description: Address folder.
+      type: String
+    - contextPath: PrismaSase.Address.id
+      description: Address ID.
+      type: String
+    - contextPath: PrismaSase.Address.address_value
+      description: Address value.
+      type: String
+    - contextPath: PrismaSase.Address.type
+      description: Address type.
+      type: String
+    - contextPath: PrismaSase.Address.name
+      description: Address name.
+      type: String
+  - arguments:
+    - default: false
+      description: A specific config job to return. If not specified, all config jobs will be returned.
+      isArray: false
+      name: job_id
+      required: false
+    - default: false
+      defaultValue: '50'
+      description: The maximum number of results to return. Default is 50.
```

```suggestion
      description: The maximum number of results to return. 
```
No need to put the default value in the description. When the README is generated it automatically picks up the default value from defaultValue.